.neutrinorc.js

```diff
@@ -0,0 +1,134 @@
+        // Remove when enabled by default (https://github.com/neutrinojs/neutrino/issues/1131).
+        overlay: true,
+        proxy: {
+          // Proxy any paths not recognised by webpack to the specified backend.
+          '*': {
+            changeOrigin: true,
+            headers: {
+              // Prevent Django CSRF errors, whilst still making it clear
+              // that the requests were from local development.
+              referer: `${BACKEND}/webpack-dev-server`,
+            },
+            target: BACKEND,
+            onProxyRes: (proxyRes) => {
+              // Strip the cookie `secure` attribute, otherwise production's cookies
+              // will be rejected by the browser when using non-HTTPS localhost:
+              // https://github.com/nodejitsu/node-http-proxy/pull/1166
+              const removeSecure = str => str.replace(/; secure/i, '');
+              const cookieHeader = proxyRes.headers['set-cookie'];
+              if (cookieHeader) {
+                proxyRes.headers['set-cookie'] = Array.isArray(cookieHeader)
+                  ? cookieHeader.map(removeSecure)
+                  : removeSecure(cookieHeader);
+              }
+            },
+          },
+        },
+        // Inside Vagrant filesystem watching has to be performed using polling mode,
+        // since inotify doesn't work with Virtualbox shared folders.
+        watchOptions: process.env.USE_WATCH_POLLING && {
+          // Poll only once a second and ignore the node_modules folder to keep CPU usage down.
+          poll: 1000,
+          ignored: /node_modules/,
+        },
+      },
+      devtool: {
+        // Enable source maps for `yarn build` too (but not on CI, since it doubles build times).
+        production: process.env.CI ? false : 'source-map',
+      },
+      style: {
+        // Disable Neutrino's CSS modules support, since we don't use it.
+        modules: false,
+      },
+      targets: {
+        browsers: [
+          'last 1 Chrome versions',
+          'last 1 Edge versions',
+          'last 1 Firefox versions',
+          'last 1 Safari versions',
+        ],
+      },
+    }],
+    ['@neutrinojs/copy', {
+      patterns: [
+        'ui/contribute.json',
+        'ui/revision.txt',
+        'ui/robots.txt',
+      ],
+    }],
+    (neutrino) => {
+      neutrino.config
+        .plugin('provide')
+        .use(require.resolve('webpack/lib/ProvidePlugin'), [{
+          // Required since AngularJS and jquery.flot don't import jQuery themselves.
+          jQuery: 'jquery',
+          'window.jQuery': 'jquery',
+        }]);
+
+      if (process.env.NODE_ENV === 'production') {
+        // Fail the build if these file size thresholds (in bytes) are exceeded,
+        // to help prevent unknowingly regressing the bundle size (bug 1384255).
+        neutrino.config.performance
+          .hints('error')
+          .maxAssetSize(1.29 * 1024 * 1024)
+          .maxEntrypointSize(1.63 * 1024 * 1024);
```

Comment:
I'm expecting that we'll need to change these max file size thresholds from time to time - but it will mean it can be considered as part of the review process, rather than increases going unnoticed when we add a larger than realised package (or a new import between our files that inadvertently pulls in too much).